### DISCUSSION TITLE
Account Blocked and can't seem to get help

### DISCUSSION THREAD

**Discussion by chrisdrackett** [vercel/community#1824] [Help] [answered]
### Summary

Last week I noticed my account somehow is in a "Blocked" state and none of my sites are accessible. This was a surprise as I never got an email letting me know this was the case.

I immediately contacted support about the issue, but the only response I got back last week was:

> It looks like the payment was unsuccessful and the account was blocked. You can fix this issue here on your dashboard

The issue is that I'm unable to select the account in question. It is grayed out in the dropdown with a large red "Blocked" tag next to the team name:

![blocked](https://media.cleanshot.cloud/media/668/HA8PcEX92raMdJ39Tw8pC94rDt6nxYuZoHAbsob4.jpeg?Expires=1679363821&Signature=XgsXF8dnwjp~Gq8mTVI4KSSEu3ht~Yq~GSdL32tDQHXH3lhfbXlPPOIox7Dw9JXyUDyaNKfQ4tZGeJqTxFktB8dBQ3--dh-rIPg4QvnsjTFtVhgQeXuVfknIMrYH8ZnLQhIquM9eNlhL-0xKkHVoNRkgl8ErD4hwBprItvKgZ2xD4ttMDDMQVJWWBNXCVwmv5K5YZCfJscTfGfDq-cdr7pIzTHP6~fv5Rmsst8d6DYcIrKeiZW3YRHXILF-HweRoKGcILpk3TugrxV3HwxJjkevHUL8SKOTD1N5~bC0mpirPZAE1BH5OFIvdJ78MRpXHgwOK5JQRejeyVMDUhEag-Q__&Key-Pair-Id=K269JMAT9ZF4GZ)

I've replied stating the above, but have not gotten a response other than the same canned message above earlier today.

I doubt I'll find help here but I'm desperate to get my websites and apps hosted on Vercel back up. This includes some development servers that I use for clients that are currently 100% down.

**Comment by amyegan** [comment, marked as answer]
Thanks for reaching out, Chris. I found your case, and it looks like the support team solved the problem for you yesterday. Please let us know if you're still having any trouble with this 🙏
